src/smallgraphs.jl

```diff
@@ -69,6 +69,29 @@ function PathDiGraph(n::Integer)
     return g
 end
 
+"""Creates a circle graph with `n` vertices. A circle graph is a closed path graph.
+"""
+function CircleGraph(n::Integer)
```

Comment:
Can we rename this to CycleGraph and CycleDiGraph? wikipedia defines a Circle Graph as https://en.wikipedia.org/wiki/Circle_graph.
Since Wikipedia is the arbiter of all knowledge...
https://en.wikipedia.org/wiki/Cycle_graph.